src/network/networkd-dhcp6.c

```diff
@@ -842,3 +963,17 @@ static int dhcp6_prefix_remove_all(Manager *m, Link *link) {
 
         return 0;
 }
+
+static bool dhcp6_link_has_dhcpv6_prefix(Link *link) {
+        Iterator i;
+        Link *l;
+
+        assert(link);
+        assert(link->manager);
+
+        HASHMAP_FOREACH(l, link->manager->dhcp6_prefixes, i) {
+                if (link == l) return true;
```

Comment:
we generally insert a newline after the if condition, before the body of the if check